libtransmission/transmission.h

```diff
@@ -715,6 +715,15 @@ tr_torrent** tr_sessionLoadTorrents(tr_session* session, tr_ctor* ctor, int* set
 ***
 **/
 
+/**
+ *  List all the torrents in current session.
+ */
+std::vector<tr_torrent*> tr_sessionGetTorrents(tr_session* session);
```

Comment:
(See comment below about using the return value  from tr_sessionLoadTorrents() instead)